### PR TITLE
docs: minor Markdown tweaks.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Lighthouse requires Chrome 54 or later.
 
 Install from the Chrome Web Store: [chrome.google.com/webstore/detail/lighthouse/…](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk)
 
-Quick-start guide on using the Lighthouse extension: http://bit.ly/lighthouse-quickstart
+Quick-start guide on using the Lighthouse extension: <http://bit.ly/lighthouse-quickstart>
```
Comment (ebidel) [inline readme.md]: I think these links are already linked up in Github. What does the `<>` do?

Comment (XhmikosR) [inline readme.md]: `<>` means it's a link explicitly. Here it works because it's GitHub Markdown and they do it automatically :)

**Comment by XhmikosR** [comment]
Added a couple more tweaks.

File: readme.md (hunk 1 above)
Comment (ebidel) [inline readme.md]: Seems unnecessary to me, but the change is harmless.

**Pull request merged**